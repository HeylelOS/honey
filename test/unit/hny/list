#!/bin/sh

set -e

# We must sort to keep consistency as hny doesn't list alphabetically
"${TEST_BIN}" list | sort

