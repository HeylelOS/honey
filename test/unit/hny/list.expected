archive
archive-1.0.0
arxiv
